pymc/step_methods/metropolis.py

```diff
@@ -140,22 +140,22 @@ def __init__(
 
         Parameters
         ----------
-        vars: list
+        vars : list
             List of value variables for sampler
-        S: standard deviation or covariance matrix
+        S : standard deviation or covariance matrix
             Some measure of variance to parameterize proposal distribution
-        proposal_dist: function
+        proposal_dist : function
             Function that returns zero-mean deviates when parameterized with
             S (and n). Defaults to normal.
-        scaling: scalar or array
+        scaling : scalar or array
             Initial scale factor for proposal. Defaults to 1.
-        tune: bool
+        tune : bool
             Flag for tuning. Defaults to True.
-        tune_interval: int
+        tune_interval : int
             The frequency of tuning. Defaults to 100 iterations.
-        model: PyMC Model
+        model : PyMC Model
```

Comment:
```suggestion
        model : Model
```

using only `Model` should trigger the use of this alias: https://github.com/pymc-devs/pymc/blob/main/docs/source/conf.py#L73 and render that as a link to the pymc.Model api documentation.

note, it might not work right now because a while back some objects were undocumented. For example, the `Model` class might not be currently documented at `pymc.Model`. If that were the case ignore it; the docs for pymc.Model will be fixed at some point and everything will work again.

Reply:
I have changed this one based on your suggestions.